my_projects.html

```diff
@@ -0,0 +1,298 @@
+              <stop offset="0.893906" stop-color="#C0BA1B" stop-opacity="0.7349" />
+            </linearGradient>
+          </defs>
+        </svg></a>
+      <div class="collapse navbar-collapse justify-content-lg-start" id="navbarNav">
+
+        <ul class="navbar-nav ml-auto">
+          <li class="nav-item">
+            <a class="nav-link mx-5 fs-4" href="index.html">Accueil</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link mx-5 fs-4" href="about.html">À Propos</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link active mx-5 fs-4" href="my_projects.html">Mes Projets</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link mx-5 fs-4" href="Contact.html">Contact</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+
+  <!-- main -->
+
+  <main class="container-fluid">
+
+    <h2 class="text-warning ms-3 mt-5">Projets Récents :</h2>
+    <div id="carouselExampleCaptions" class="carousel slide mt-5">
+      <div class="carousel-indicators mt-5">
+        <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="0" class="active"
+          aria-current="true" aria-label="Slide 1"></button>
+        <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="1"
+          aria-label="Slide 2"></button>
+        <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="2"
+          aria-label="Slide 3"></button>
+        <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="3"
+          aria-label="Slide 4"></button>
+        <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="4"
+          aria-label="Slide 5"></button>
+        <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="5"
+          aria-label="Slide 6"></button>
+      </div>
+      <div class="carousel-inner">
+
+        <div class="carousel-item active mb-3 ">
+          <div class="card border-warning mb-5 w-75" style="margin: 0% auto;">
+            <img src="images/Site_imdb.jpg" class="card-img-top" alt="IMDB">
+            <div class="card-group bg-dark p-3 border border-warning  text-white">
+              <h3 class="card-title "> • Site IMDB</h3>
+              <p class="card-text fs-3 p-3">- Site IMDB créé avec HTML et CSS, avec cinq pages et une page de détails et
+                valeur,
+                Le site web Netflix permet aux utilisateurs de regarder des films en ligne via leur ordinateur, leur
+                téléphone portable,
+                leur tablette ou leur télévision connectée.
+                Les utilisateurs peuvent choisir parmi une sélection de titres dans différentes catégories telles que
+                les films d'action,
+                les comédies, les drames, les documentaires, et bien plus encore.</p>
```

Comment:
Why did you have a lot of breaks here for your text? You could put all your text together. The same for your other projects.